go/hyperkit.go

```diff
@@ -82,8 +87,8 @@ type HyperKit struct {
 	VPNKitKey string `json:"vpnkit_key"`
 	// UUID is a string containing a UUID, it sets BIOS DMI UUID for the VM (as found in /sys/class/dmi/id/product_uuid on Linux).
 	UUID string `json:"uuid"`
-	// DiskImage is the path to the disk image to use
-	DiskImage string `json:"disk"`
+	// DiskImages contains disk images to use/create.
+	DiskImages []DiskImage `json:"disk_images"`
```

Comment:
Maybe make this: `Disks []DiskConfig` like here: https://github.com/linuxkit/linuxkit/blob/master/src/cmd/linuxkit/run_qemu.go#L27